account.html

```diff
@@ -0,0 +1,166 @@
+          </ul>
+        </nav>
+        <a href="./cart.html"
+          ><img src="./images/cart.png" alt="" width="30px" height="30px"
+        /></a>
+        <img
+          src="./images/menu.png"
+          alt=""
+          width="30px"
+          height="30px"
+          class="menu-icon"
+          onclick="menutoggle()"
+        />
+      </div>
+    </div>
+
+    <!--account-->
+
+    <div class="account-page">
+      <div class="container">
+        <div class="row">
+          <div class="col-2">
+            <img src="./images/image1.png" alt="" />
+          </div>
+          <div class="col-2">
+            <div class="form-container">
+              <div class="form-btn">
+                <span onclick="login()">Login</span>
+                <span onclick="register()">Register</span>
+                <hr id="Indicator" />
+              </div>
+              <form id="LoginForm">
+                <input type="text" placeholder="username" />
+                <input type="password" placeholder="password" />
+                <button class="btn" type="submit">login</button>
+                <a href="#">forget password</a>
+              </form>
+              <form id="RegForm">
+                <input type="text" placeholder="Username" />
+                <input type="email" placeholder="Email" />
+                <input type="password" placeholder="Password" />
+                <button class="btn" type="submit">Register</button>
+              </form>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <!--Footer-->
+
+    <div class="footer">
+      <div class="container">
+        <div class="row">
+          <div class="footer-col-1">
+            <h3>Download our app</h3>
+            <p>Download our app for android and ios</p>
+            <div class="app-logo">
+              <img src="./images/play-store.png" alt="" />
+              <img src="./images/app-store.png" alt="" />
+            </div>
+          </div>
+          <div class="footer-col-2">
+            <img src="./images/logo-white.png" alt="" />
+            <p>
+              our purpose is to purposefull make the please and bengits of
+              sports accesible
+            </p>
+          </div>
+          <div class="footer-col-3">
+            <h3>useful links</h3>
+            <ul>
+              <li>coupons</li>
+              <li>blogposts</li>
+              <li>return policy</li>
+              <li>join affiliate</li>
+            </ul>
+          </div>
+          <div class="footer-col-4">
+            <h3>follow us</h3>
+            <ul>
+              <li>facebook</li>
+              <li>twitter</li>
+              <li>rinstagram</li>
+              <li>youtube</li>
+            </ul>
+          </div>
+        </div>
+        <hr />
+        <p class="copyright">copyright 2020 amazon.com</p>
+      </div>
+    </div>
+
+    <script>
+      var MenuItems = document.getElementById('MenuItems');
```

Comment:
Adding a separate js file for code would be better

Reply:
sure, will add